### PR TITLE
[WIP] stegoveritas

### PR DESCRIPTION
**Pull request by noraj**
issue #2302

### CODE REVIEW

File: packages/stegoveritas/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stegoveritas
+pkgver=1.1
+pkgrel=1
+pkgdesc='Automatic image steganography analysis tool'
+arch=('any')
+groups=('blackarch' 'blackarch-stego')
+url='https://github.com/bannsec/stegoVeritas/releases'
```
Comment (Edu4rdSHL) [inline packages/stegoveritas/PKGBUILD]: Please, remove the "releases" string.

File: packages/stegoveritas/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+license=('MIT')
+depends=('python' 'python-pillow' 'python-numpy' 'python-python-magic' 'python-prettytable' 'python-exifread' 'python-python-xmp-toolkit' 'python-stegoveritas-binwalk' 'python-pypng' 'python-apng' 'perl-image-exiftool' 'p7zip' 'foremost')
```
Comment (Edu4rdSHL) [inline packages/stegoveritas/PKGBUILD]: Try to indent better (no all in one line)

File: packages/stegoveritas/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+makedepends=('python-setuptools')
+source=("$pkgname-$pkgver.tar.gz::https://github.com/bannsec/stegoVeritas/archive/$pkgver.tar.gz")
```
Comment (Edu4rdSHL) [inline packages/stegoveritas/PKGBUILD]: Just use https://github.com/bannsec/stegoVeritas/archive/$pkgver.tar.gz

File: packages/stegoveritas/PKGBUILD (hunk 1 above)
Comment (noraj) [reply packages/stegoveritas/PKGBUILD]: So maybe the description `URL pointing to download section of package` from the template should be change to `URL pointing to the source code or download section of package` https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python

File: packages/stegoveritas/PKGBUILD (hunk 2 above)
Comment (noraj) [reply packages/stegoveritas/PKGBUILD]: ok

File: packages/stegoveritas/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+license=('MIT')
+depends=('python' 'python-pillow' 'python-numpy' 'python-python-magic' 'python-prettytable' 
+         'python-exifread' 'python-python-xmp-toolkit' 'python-stegoveritas-binwalk' 'python-pypng'
+         'python-apng' 'perl-image-exiftool' 'p7zip' 'foremost')
+makedepends=('python-setuptools')
+source=("https://github.com/bannsec/stegoVeritas/archive/$pkgver.tar.gz")
+sha512sums=('9b112f803f00bba1ca711dffcae5d8358022a5b38a19dbd46531ba48b8e374e2b6f87d1e5e45fd35e542f7fc6df662f78c21887107b5615ac4afe5d693b25ae6')
+
+build() {
+  cd "$pkgver"
```
Comment (Edu4rdSHL) [inline packages/stegoveritas/PKGBUILD]: The result of the extract is a dir with name `stegoVeritas-1.1`, no `1.1`, please fix that in all the `cd`.

File: packages/stegoveritas/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stegoveritas
```
Comment (Edu4rdSHL) [inline packages/stegoveritas/PKGBUILD]: You can add here a `_pkgname=stegoVeritas` in order to use `$_pkgname-$pkgver` when doing the `cd` in build() and package().

**Comment by Edu4rdSHL** [comment]
Thank you @noraj, now LGTM, merging.

**Pull request merged**